chainstate/src/detail/chainstateref/tx_verifier_storage.rs

```diff
@@ -86,6 +90,15 @@ impl<'a, S: BlockchainStorageRead, V: TransactionVerificationStrategy> Transacti
             .map_err(TransactionVerifierStorageError::from)
     }
 
+    fn get_tokens_accounting_undo(
```

Comment:
Not sure whether we should always do this, but maybe marking functions relating to tokens v2 is important to make the distinction.

Reply:
I don't think it's necessary. I think this methods should be version agnostic (even though they potentially can return versioned undo objects in the future if we implement that). Current we don't even have to make undo versioned because it's not a part of a block and everything in the db we just reindex.